frame/support/src/traits/tokens/nonfungible_v2.rs

```diff
@@ -24,6 +24,7 @@
 //! For an NFT API that has dual-level namespacing, the traits in `nonfungibles` are better to
 //! use.
 
+use super::misc::ItemId;
```

Comment:
I would add the `ItemId, CollectionId and DestroyWitness` into the `pub use misc::{` of the `frame/support/src/traits/tokens.rs` and import later via `use frame_support::traits::tokens::{CollectionId, ItemId}`

Reply:
Can you help? I think this PR can be closed later

Reply:
Just open the `frame/support/src/traits/tokens.rs` file, find `pub use misc::{` and add there `ItemId, CollectionId, DestroyWitness`

Reply:
Done